lesson0.md

```diff
@@ -1,15 +1,64 @@
-![](https://img.shields.io/badge/status-draft-darkred.svg)
+![](https://img.shields.io/badge/status-review-orange.svg)
 
 # Lesson 0
-** What we will learn today?**
-- [Git](#Git)
-- [IDEs](#IDE)
-- [Terminal](#Terminal-basics)
-- [CodeYourFuture](#CodeYourFuture)
+**What we will learn today?**
+- [Terminal](#terminal-basics)
+- [Git](#git)
+- [IDEs](#ide)
+- [CodeYourFuture](#codeyourfuture)
+- [Homework](#homework)
 ---
 
+## Terminal basics
+
+Welcome to the world of Web Development! During your course journey and onwards, the
+**terminal** (also known as **command line**) will be one of your most valuable tools.
+It will help you to interact with you computer faster, by helping you to:
+
+- Create and delete files.
+- Install web development tools.
+- Start a server.
+- Use source control (explained [later in this lesson](#Git)).
+
+All this will make sense as we progress with the lessons. For now, we would like you
+to get familiar with it by following this tutorial:
+
+- [Learn the command line >](https://www.codecademy.com/learn/learn-the-command-line)
+
+### How do I use it on my computer?
+
+So now that you know what the terminal can do for you, how do you use it? Please find
+instructions for your operating system below. After you have followed the instructions,
+open a terminal and write `ls`, then press the `Enter` key. You will see something
+like this:
+
+<div style="text-align: center;">
+  <img
+    src="assets/lesson-0/terminal.png"
+    width="300"
+    alt="Terminal"
+  />
+</div>
+
+#### Linux
+
+The Terminal is already installed. It should be in the list of applications available in
+your computer. [More instructions >](http://www.wikihow.com/Open-a-Terminal-Window-in-Ubuntu)
+
+#### Mac OSx (Apple)
+
+The Terminal is already installed. It should be in the list of applications available in
+your computer. [More instructions >](http://www.wikihow.com/Get-to-the-Command-Line-on-a-Mac)
+
+#### Windows
+
+Install [cmder](http://cmder.net/). After that, open it from your windows menu. When
```

Comment:
not sure - I was thinking Git Bash would do on Windows. I think cmder has a bash shell?